### ISSUE TITLE
JAWS fails to recognize newly focused table cells when changed with Arrow keys

### ISSUE THREAD

**Issue by sclower** [FreedomScientific/VFO-standards-support#871]
### Summary

JAWS doesn't alert users to newly focused table cell content when focus is programmatically set in response to Arrow key presses.

### Expected result

JAWS announces the newly focused cell, speaks its column or row (depending on user verbosity preferences), along with the focused textarea; the same that would normally occur when navigating by Tab and Shift+Tab.

### Actual result

Silence.

### Example

1.  Open [this JSBin](https://jsbin.com/veguxexice/edit?html,output).
2. Tab to any of the textarea elements within the table. Ensure Forms Mode is on.
3. Use Up, Down, Left, and Right Arrows to cause the Javascript to focus the cell in the corresponding direction.

### Additional Information

We use textareas inside dynamic tables in [the Desmos graphing calculator](https://www.desmos.com/calculator), and JAWS users are having a hard time using it. NVDA, VoiceOver, and Talkback are working as expected.

#### JAWS version and build number

2024.2403.3

 
#### Operating System and version

Windows 11 23H2 (OS Build 22631.4037)

#### Browser and version:

Chrome 129, Firefox 128

**Comment by JAWS-test** [comment]
Can't reproduce the problem. I have tested with the latest Chrome and JAWS 2024. When I navigate to the table cells in form mode, I hear the content correctly. 

As inside the table cells are textarea elements, JAWS does not output their contents, but only whether they are empty or contain text. However, this is normal for textarea (even outside a table)

Please note that arrow key navigation is not supported for a table. In this case, role=grid should be used. See https://www.w3.org/WAI/ARIA/apg/patterns/table/ and https://www.w3.org/WAI/ARIA/apg/patterns/grid/

**Comment by sclower** [comment]
Thank you for looking into this issue. I also prefer using the grid 
pattern when practical, but historically JAWS hasn't worked well with it 
when we last checked. Regardless, I found identical behavior when 
implementing it, and I won't rule out that some setting in my 
installation is bringing out a problem which you aren't seeing. To be 
clear, Tabbing through cells inside Forms Mode works in either case. Can 
you confirm that Arrow keys also cause JAWS to report the new position 
of your focus, again while remaining in Forms Mode?

**Comment by JAWS-test** [comment]
> Can you confirm...

Yes

**Comment by sclower** [comment]
@JAWS-test I spoke with a TVI recently who is also seeing the same behavior. If you are unable to duplicate, I'd appreciate details on exactly what you tried. So far I'm able to reproduce the problem even in JAWS 2026.
